docs/rust-ci-cd-integration.md

```
@@ -0,0 +1,435 @@
+	@echo "Testing Rust support..."
+	GOCLEAN_TEST_MODE=1 $(GOTEST) -v ./internal/scanner -run="Test.*Rust"
+	GOCLEAN_TEST_MODE=1 $(GOTEST) -v ./internal/violations -run="Test.*Rust"
+
+# Run Rust benchmarks
+benchmark-rust:
+	@echo "Running Rust benchmarks..."
+	$(GOTEST) -bench=BenchmarkRust -benchmem ./internal/scanner
+
+# Validate Rust integration
+validate-rust: test-rust benchmark-rust
+	@echo "Creating Rust test project..."
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)